runtime/v2/runner/src/main/java/com/walmartlabs/concord/runtime/v2/runner/el/resolvers/SensitiveDataProcessor.java

```diff
@@ -53,11 +56,41 @@ public static void process(Object value, Method method) {
                 Object v = m.get(key);
                 if (v instanceof String) {
                     SensitiveDataHolder.getInstance().add((String) v);
+                } else if (a.includeNestedValues() && v instanceof Map<?,?> nested) {
+                    processNestedValues(nested, 0);
                 }
             }
         }
     }
 
+    private static void processNestedValues(Object obj, int depth) {
+        if (depth > MAX_DEPTH) {
+            return;
+        }
+
+        if (obj instanceof Map<?, ?> map) {
+            for (Map.Entry<?, ?> entry : map.entrySet()) {
+                processNestedValues(entry.getValue(), depth + 1);
+            }
+        } else if (obj instanceof List<?> list) {
+            for (Object o : list) {
+                processNestedValues(o, depth + 1);
+            }
+        } else if (obj instanceof Set<?> set) {
+            for (Object item : set) {
+                processNestedValues(item, depth + 1);
+            }
+        } else if (obj != null && obj.getClass().isArray()) {
+            var len = Array.getLength(obj);
```

Comment:
No mixing var/no-var in the same file, please.

Reply:
fixed!